guides/doc-Managing_Hosts/topics/proc_registering-a-host.adoc

```diff
@@ -1,14 +1,14 @@
-[id="registering-a-host-to-project-using-the-global-registration-template_{context}"]
-= Registering a Host to {ProjectName} Using the Global Registration Template
+[id="registering-a-host_{context}"]
+= Registering a Host to {ProjectName}
```

Comment:
For the record, this PR changed the ID however the code still contained references:

```
$ ag "registering-a-host-to-project-using-the-global-registration-template"
guides/common/assembly_configuring-and-setting-up-remote-jobs.adoc
32:For more information, see {ManagingHostsDocURL}registering-a-host-to-project-using-the-global-registration-template_managing-hosts[Registering a Host to {ProjectName} Using the Global Registration Template].

guides/doc-Content_Management_Guide/topics/QuickStart.adoc
154:Use your activation key to register a host to the Content View. For more information on registering a host, see {ManagingHostsDocUR}registering-a-host-to-project-using-the-global-registration-template_managing-hosts

guides/doc-Managing_Hosts/topics/Registering_Hosts.adoc
25:* xref:registering-a-host-to-project-using-the-global-registration-template_managing-hosts[]

guides/doc-Managing_Hosts/topics/using_insights_with_satellite_hosts.adoc
12:For more information, see {ManagingHostsDocURL}registering-a-host-to-project-using-the-global-registration-template_managing-hosts[Registering a Host to {ProjectName} Using the Global Registration Template] in the _Managing Hosts_ guide.
```

I wonder how was possible that our HTML validation passed. I mean, those that has links to the site surely passed, this is something we need to improve, but I would expect `xref:registering-a-host-to-project-using-the-global-registration-template_managing-hosts[]` to fail.